tests/main/prepare-image-validation-sets/task.yaml

```diff
@@ -0,0 +1,162 @@
+summary: Verify that we are able to use validation sets with the model assertion through prepare-image 
+
+# autopkgtest run only a subset of tests that deals with the integration
+# with the distro
+backends: [-autopkgtest]
+
+# disable the following distributions
+# ubuntu-14, lack of systemd-run
+# ubuntu-18.04-32: pc gadget from 20 is not available
+# ubuntu-20.04-arm*, because we use pc kernel and gadget.
+systems:
+- -ubuntu-14.04-*
+- -ubuntu-18.04-32
+- -ubuntu-20.04-arm-*
+
+environment:
+    ROOT: /home/test/tmp/
+    IMAGE: /home/test/tmp/system-seed
```

Comment:
it can be
IMAGE: $ROOT/system-seed
GADGET: $ROOT/gadget

Reply:
Added!